<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="3" ignored="0" total="7" passed="4">
  <reporter-output>
  </reporter-output>
  <suite name="FacebookApp Suite" duration-ms="6924" started-at="2017-12-11T14:37:05Z" finished-at="2017-12-11T14:37:12Z">
    <groups>
    </groups>
    <test name="FacebookApp test" duration-ms="6924" started-at="2017-12-11T14:37:05Z" finished-at="2017-12-11T14:37:12Z">
      <class name="com.testprojects.facebook.tests.FacebookRatingTest">
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.testprojects.facebook.tests.FacebookRatingTest@6ad5c04e]" name="setUp" is-config="true" duration-ms="2228" started-at="2017-12-11T16:37:03Z" finished-at="2017-12-11T16:37:05Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="cleanUp()[pri:0, instance:com.testprojects.facebook.tests.FacebookRatingTest@6ad5c04e]" name="cleanUp" is-config="true" duration-ms="683" started-at="2017-12-11T16:37:05Z" finished-at="2017-12-11T16:37:05Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- cleanUp -->
        <test-method status="PASS" signature="searchOnGoogle()[pri:0, instance:com.testprojects.facebook.tests.FacebookRatingTest@6ad5c04e]" name="searchOnGoogle" duration-ms="2218" started-at="2017-12-11T16:37:05Z" finished-at="2017-12-11T16:37:08Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- searchOnGoogle -->
        <test-method status="PASS" signature="clickOnSearchResult()[pri:0, instance:com.testprojects.facebook.tests.FacebookRatingTest@6ad5c04e]" name="clickOnSearchResult" duration-ms="3280" started-at="2017-12-11T16:37:08Z" depends-on-methods="com.testprojects.facebook.tests.FacebookRatingTest.searchOnGoogle" finished-at="2017-12-11T16:37:11Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- clickOnSearchResult -->
        <test-method status="PASS" signature="test1()[pri:0, instance:com.testprojects.facebook.tests.FacebookRatingTest@6ad5c04e]" name="test1" duration-ms="443" started-at="2017-12-11T16:37:11Z" depends-on-methods="com.testprojects.facebook.tests.FacebookRatingTest.clickOnSearchResult" finished-at="2017-12-11T16:37:11Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test1 -->
        <test-method status="FAIL" signature="test2()[pri:0, instance:com.testprojects.facebook.tests.FacebookRatingTest@6ad5c04e]" name="test2" duration-ms="100" started-at="2017-12-11T16:37:11Z" depends-on-methods="com.testprojects.facebook.tests.FacebookRatingTest.clickOnSearchResult" finished-at="2017-12-11T16:37:12Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[4 > 3, 3 > 2 , 2 > 1 expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 4 > 3, 3 > 2 , 2 > 1 expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at com.testprojects.facebook.pages.FacebookGooglePlayPage.fourStarsAreMoreThan3AreMoreThan2StarsAreMoreThan1Star(FacebookGooglePlayPage.java:53)
	at com.testprojects.facebook.tests.FacebookRatingTest.test2(FacebookRatingTest.java:47)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:72)
	at org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:123)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test2 -->
        <test-method status="FAIL" signature="test3()[pri:0, instance:com.testprojects.facebook.tests.FacebookRatingTest@6ad5c04e]" name="test3" duration-ms="65" started-at="2017-12-11T16:37:12Z" depends-on-methods="com.testprojects.facebook.tests.FacebookRatingTest.clickOnSearchResult" finished-at="2017-12-11T16:37:12Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[3 > 2, 2 > 1 expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 3 > 2, 2 > 1 expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at com.testprojects.facebook.pages.FacebookGooglePlayPage.threeStarsAreMoreThan2StarsAreMoreThan1Star(FacebookGooglePlayPage.java:63)
	at com.testprojects.facebook.tests.FacebookRatingTest.test3(FacebookRatingTest.java:56)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:72)
	at org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:123)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test3 -->
        <test-method status="FAIL" signature="test4()[pri:0, instance:com.testprojects.facebook.tests.FacebookRatingTest@6ad5c04e]" name="test4" duration-ms="48" started-at="2017-12-11T16:37:12Z" depends-on-methods="com.testprojects.facebook.tests.FacebookRatingTest.clickOnSearchResult" finished-at="2017-12-11T16:37:12Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[2 stars should be more than 1 star expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 2 stars should be more than 1 star expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at com.testprojects.facebook.pages.FacebookGooglePlayPage.twoStarsAreMoreThanOneStar(FacebookGooglePlayPage.java:70)
	at com.testprojects.facebook.tests.FacebookRatingTest.test4(FacebookRatingTest.java:65)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:72)
	at org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:123)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test4 -->
        <test-method status="PASS" signature="test5()[pri:0, instance:com.testprojects.facebook.tests.FacebookRatingTest@6ad5c04e]" name="test5" duration-ms="61" started-at="2017-12-11T16:37:12Z" depends-on-methods="com.testprojects.facebook.tests.FacebookRatingTest.clickOnSearchResult" finished-at="2017-12-11T16:37:12Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test5 -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.testprojects.facebook.tests.FacebookRatingTest@6ad5c04e]" name="tearDown" is-config="true" duration-ms="160" started-at="2017-12-11T16:37:12Z" finished-at="2017-12-11T16:37:12Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.testprojects.facebook.tests.FacebookRatingTest -->
    </test> <!-- FacebookApp test -->
  </suite> <!-- FacebookApp Suite -->
</testng-results>
